ocs_ci/ocs/constants.py

```diff
@@ -39,15 +39,20 @@
 PVC = "PersistentVolumeClaim"
 POD = "Pod"
 
+
 # Other
 SECRET = "Secret"
 NAMESPACE = 'Namespace'
 IGNORE_SC = "gp2"
-
+DEFAULT_BLOCKPOOL = 'rbd'
```

Comment:
Is this a default or a constant? If it will ever be overwritten let's move it to `defaults.py` If it isn't, let's remove `DEFAULT_` from the name.

Reply:
@clacroix12 Let's address this in a different PR in order not to hold this one as it's open for a long time.
@keesturam Can you please submit one after this one is merged?